src/azure-cli-core/azure/cli/core/auth/util.py

```diff
@@ -58,10 +58,16 @@ def _generate_login_command(scopes=None, claims_challenge=None):
 
     # Rejected by Conditional Access policy, like MFA
     if scopes:
-        login_command.append('--scope {}'.format(' '.join(scopes)))
+        login_command.append('--scope')
+        for s in scopes:
+            login_command.append(f'"{s}"')
```

Comment:
These lines can be combined using [* unpacking](https://peps.python.org/pep-0448/), but the readability will be bad:

```py
login_command.extend(['--scope', *(f'"{s}"' for s in scopes)])
```